src/app/components/Document/index.jsx

```diff
@@ -18,12 +18,16 @@ const Document = ({ assets, data, styleTags, helmet }) => {
       <head>
         <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
         <meta charSet="utf-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta
```

Comment:
We shouldn't really be setting any of these meta tags in Document, we should set them in a single place, in `Article`. Just move this `viewport` one for now, and can open another PR to move all of them at another time.

Reply:
Thanks for your comments. Will make the changes now.